[Fix] (Workbox wizard) Manually enter the path of the web app root and unselectable separator

ognjenjevremovic opened the pull request:
Fixes *#2764*

R: @jeffposnick @philipwalton

### Changes
* Fixed error thrown during the workbox wizard regarding the web application root directory prompt (first question) when `Manually enter path` option is chosen,
* Made the separator unselectable,
* Extended upon the existing unit tests regarding this functionality.

### Checklist
After the changes I ensured that:
 * no stylistic or unwanted errors are present, by running `npm run gulp lint` from the root directory,
 * no tests are failing, by running `npm run gulp test_node` from the root directory,
 * build passes with no issues, by running `npm run gulp build` from the root directory,
 * checked the build output locally by:
   * creating a new `npm` project and installing the `workbox` (local) package, by running `npm i ~/path/to/local/project/workbox/packages/workbox-cli`,
   * spawning the workbox wizard from the project, by running `./node_modules/.bin/workbox wizard`,
   * checked the functionality manually with cases covered:
     1. no child directories present,
     2. multiple child directories present,
     3. manual input of existing directory,
     4. manual input of non-existing directory.
   * checking the generated `workbox-config.js` file content after completing the wizard.

### Done
- [x] Added/updated unit tests for this change
- [x] Included links to related issues/PRs

Commit:
fix: 🐛 workbox-cli ask root of web app question

Make separator unselectable as an answer to root of web application
question when multiple child directories are present. Fix the manual
input to root of the web application question - when manual input option
is chosen a new question / input will be provided to the user to specify
the root of the web app.

✅ Closes: #2764

Commit:
test: 💍 workbox-cli ask root of web app question

Extend upon the existing workbox-cli ask root of web app question with
additional unit tests.

✅ Closes: #2764

jeffposnick commented:
Thank you for this! I haven't had time to test the changes locally this week, but I wanted to let you know that it hasn't slipped my mind.

ognjenjevremovic commented:
No worries. Thank you for getting back to me. 
Please do let me know if there are any changes required prior to merging this, once you get time to review the PR.

I'm closely following the `workbox` project as I familiarized myself with the codebase over the past week(s).
Although, there aren't as many open issues (especially regarding the `cli` part that I'm most familiar with) I'll be sure to follow the progress and bug reports and try to contribute to the project as much as I can.
Also, I'll try to keep up with the rest of the amazing OSS tools and projects you guys are making freely available for the rest to use and contribute as I see fit.

Thanks and best regards! 🙂

jeffposnick commented on packages/workbox-cli/src/lib/questions/ask-root-of-web-app.ts:
```diff
@@ -17,8 +17,9 @@ import {constants} from '../constants';
-// The key used for the question/answer.
-const name = 'globDirectory';
+// The keys used for the questions/answers.
+const question_rootDirectory = 'globDirectory';
```
Can you switch this and the other variable to `questionRootDirectory` instead of `question_rootDirectory`?

That's more in keeping with our variable naming pattern.

ognjenjevremovic replied:
Hey @jeffposnick, thank you for going through the PR and reporting back.
I'll be sure to address the changes asap 🙂 .

jeffposnick commented on test/workbox-cli/node/lib/questions/ask-root-of-web-app.js:
```diff
@@ -14,8 +14,12 @@ const {errors} = require('../../../../../packages/workbox-cli/build/lib/errors')
 // This is the hardcoded name of the question that's passed to inquirer.
 // It's used as the key to read the response from the answer.
-const QUESTION_NAME = 'globDirectory';
+const question_rootDirectory = 'globDirectory';
```
Can you switch this and the other variable to `questionRootDirectory` instead of `question_rootDirectory`?

That's more in keeping with our variable naming pattern.

jeffposnick reviewed (changes requested):
Thank you! There are just some naming issues to resolve.

Commit:
chore: 🤖 workbox-cli root web app question variable naming

workbox-cli package, rename `root of the web application` question
variable names to use camelcase naming pattern instead of snake case
format.

✅ Closes: #2766

jeffposnick reviewed (approved):
Thanks again! We'll cut a new release that includes these fixes within a week or so.
